index.js

```diff
@@ -29,6 +29,9 @@ app.use("/fonts", express.static("./assets/fonts"))
 app.use(bodyParser.json())
 app.use(bodyParser.urlencoded({ extended: true }))
 app.use(sharify)
+// Inject METAPHYSICS_ENDPOINT to sharify data
+// Todo/Enhancement: inject any 'safe ENV vars?
+sharify.data.METAPHYSICS_ENDPOINT = process.env.METAPHYSICS_ENDPOINT
```

Comment:
changing the Relay config to using sharify meant that I needed to inject this to sharify, not in the loyalty app layer (b/c this is going to be used by components all over), but earlier in the lifecycle for the app.

It's worth noting that when this is embedded in Force, this file won't _actually_ inject the env var since we just export app (+ components), however Force does something [similar](https://github.com/artsy/force/blob/755dfe70534eb995076c60e51800aea426d32714/lib/setup_sharify.js), so as long we keep the same env var names (`API_URL` vs `ARTSY_URL`!), we should be fine using this within Force.